### PR TITLE
Digard Network Test - DIGA

### PR DESCRIPTION
**Pull request by kemal-arbeiter**

### CODE REVIEW

**Comment by rajranjan0608** [comment]
Hi @kemal-arbeiter. Can you also put a link to the indexed explorer in the config file?

**Comment by kemal-arbeiter** [comment]
done, please check

**Comment by rajranjan0608** [comment]
Thank you very much. This LGTM.

File: config.json [hunk 1]
```diff
@@ -122,6 +122,22 @@
...
                 "WINDOW_SIZE": 1440
             }
+        },
+        {
+            "ID": "DIGARD",
+            "NAME": "Digard Network Test",
+            "TOKEN": "DIGA",
+            "RPC": "https://api.digard.network/ext/bc/sNUvM8CDbYbwi99ih238cbwyRiDYsX6GPKnFF1NzignfKfte4/rpc",
+            "CHAINID": 960006,
+            "EXPLORER": "https://testnet.avascan.info/blockchain/digard/home",
```
Comment (rajranjan0608) [inline config.json]: Can you remove `home` from the explorer URL? Avascan now supports redirection from the base endpoint.

File: config.json [hunk 2]
```diff
@@ -122,6 +122,22 @@
...
+            "IMAGE": "https://cdn.digard.io/digard.jpg",
```
Comment (rajranjan0608) [inline config.json]: Would you like to use a PNG or image with a transparent background here? I am ok with both...

**Review by rajranjan0608 — CHANGES_REQUESTED** [review]
Requested a few nits. Thanks

File: config.json (hunk 1 above)
Comment (kemal-arbeiter) [reply config.json]: fixed

File: config.json (hunk 2 above)
Comment (kemal-arbeiter) [reply config.json]: fixed

**Comment by rajranjan0608** [comment]
Awesome 🤝

**Review by rajranjan0608 — APPROVED** [review]

**Comment by rajranjan0608** [comment]
@kemal-arbeiter Make sure to send funds to one and only faucet address -  0x2352D20fC81225c8ECD8f6FaA1B37F24FEd450c9

**Pull request merged**